docs/src/modules/develop/pages/gke.adoc

```diff
@@ -0,0 +1,53 @@
+= Run on GKE
+
+include::ROOT:partial$attributes.adoc[]
+include::ROOT:partial$include.adoc[]
+
+== Cluster creation
+Create a GKE cluster. We recommend at least 6 vCPUs (ie, a node pool of 3 `n1-standard-2` nodes). Also ensure that the current user is a cluster admin.
+
+Detailed instructions for creating the GKE cluster can be found in the Knative documentation, follow all the steps up to (but not including) installing Knative.
+
+== Cassandra Set up
+If using an event sourced entity, install Cassandra. This can be done from the Google Marketplace, by visiting the Cassandra Cluster, selecting configure, selecting your GCloud project, and then installing it in the Kubernetes cluster you just created.
+
+The defaults should be good enough, in our examples we called the app instance name *cassandra*.
+
+_Note: there is an option to use an in memory store if you just want to test it out, of course, as soon as your pods shut down (or if they are rebalanced), your store will be lost._
+
+== Cluster preparation
+Create the cloudstate namespace and install the Cloudstate operator:
+
+[source,sh,subs="attributes+"]
+----
+kubectl create namespace cloudstate
+kubectl apply -n cloudstate -f https://raw.githubusercontent.com/cloudstateio/cloudstate/v{cloudstate-version}/operator/cloudstate.yaml
+----
+
+== Deploying function
+You are now ready to install an event sourced function. We have a shopping cart example in the _samples/js-shopping-cart_ directory of this project. This can be installed by following these instructions:
```

Comment:
Let's use code for anything like directory names:

```suggestion
You are now ready to install an event sourced function. We have a shopping cart example in the `samples/js-shopping-cart` directory of this project. This can be installed by following these instructions:
```